crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -54,6 +56,27 @@ pub struct ParsedQObject {
 }
 
 impl ParsedQObject {
+    #[cfg(test)]
+    pub fn mock_parsed_qobject() -> Self {
+        ParsedQObject {
+            base_class: None,
+            name: Name::new(format_ident!("my_property")),
+            rust_type: format_ident!("i32"),
```

Comment:
This doesn't really make sense, you're mocking a parsed QObject, not a property.
So that should be `MyObject` and `MyObjectRust`.

Reply:
Done